conf/modules.config

```diff
@@ -72,19 +72,24 @@ process {
         ]
     }
 
-    withName: 'BWAMEM2_MEM' {
+    // This BWAMEM2_MEM belongs to the decontamination module
+    withName: 'BWAMEM2_MEM_DECONT' {
         cpus   = { check_max( 12    * task.attempt, 'cpus'    ) }
         memory = { check_max( 72.GB * task.attempt, 'memory'  ) }
-        time   = { check_max( 16.h  * task.attempt, 'time'    ) }
+        time   = { check_max( 1.h  * task.attempt,  'time'    ) }
```

Comment:
I'd double check if it can take longer than an hour to decontaminate >100 M paired-end raw-reads vs human-phiX reference

Reply:
I'll revet this one